frontend/docs/style-guide.md

```diff
@@ -42,6 +42,10 @@
   - Angular code snippets
     - [Fetch data from API: Infinite scroll](https://codepad.co/snippet/b27gCGTJ)
 
+# Indentation
+
+For all of our code we use 2 spaces of indentation. You can set it as default in your text editor settings to use tab instead with 2 spaces. 
```

Comment:
I'd change that text to:
```
We use 2 space indentation in all our files. Set your text editor to use spaces instead of tabs for indentation and set the indentation size to 2.
```

Reply:
On it!